### PR TITLE
Added example to docstring of outlier_remover

### PR DESCRIPTION
**Pull request by anopsy**
Before working on a large PR, please check with @koaning or @MBrouns that they agree with the direction of the PR. This discussion should take place in a Github issue before working on the PR, unless it's a minor change like spelling in the docs. 

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] My code follows the style guidelines (flake8)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (also to the readme.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added tests to check whether the new feature adheres to the sklearn convention
- [ ] New and existing unit tests pass locally with my changes

If you feel your PR is ready for a review, ping @koaning or @mbrouns.

### CODE REVIEW

File: sklego/preprocessing/outlier_remover.py [hunk 1]
```diff
@@ -61,6 +79,23 @@ def transform_train(self, X):
         -------
         np.ndarray of shape (n_not_outliers, n_features)
             The data with the outliers removed, where `n_not_outliers = n_samples - n_outliers`.
+	Example
```
Comment (koaning) [inline sklego/preprocessing/outlier_remover.py:82]: Minor indentation thing.

**Review by koaning — COMMENTED** [review]
Looks pretty good. Just to check: did you have a look at what it looks like when you render it?

File: sklego/preprocessing/outlier_remover.py [hunk 2]
```diff
@@ -42,6 +42,24 @@ def fit(self, X, y=None):
...
+        X = np.random.randn(10000, 2)
+
+        isolation_forest = IsolationForest()
+        isolation_forest.fit(X)
+        detector_preds = isolator_forest.predict(X)
+
+        outlier_remover = OutlierRemover(isolation_forest, refit=True)
+        outlier_remover.fit(X)
+        X_trans = outlier_remover.transform_train(X)
+        ```
```
Comment (koaning) [inline sklego/preprocessing/outlier_remover.py:62]:
```suggestion
        Example
        -------
        ```py
        from sklego.preprocessing import OutlierRemover
        from sklearn.ensemble import IsolationForest

        np.random.seed(0)
        X = np.random.randn(10000, 2)

        isolation_forest = IsolationForest()
        isolation_forest.fit(X)
        detector_preds = isolator_forest.predict(X)

        outlier_remover = OutlierRemover(isolation_forest, refit=True)
        outlier_remover.fit(X)
        ```
```

**Review by koaning — APPROVED** [review]

File: sklego/preprocessing/outlier_remover.py (hunk 1 above)
Comment (FBruzzesi) [reply sklego/preprocessing/outlier_remover.py:81]: Newline may be required?

**Pull request merged**